Cargo.toml

```
@@ -178,7 +178,7 @@ econtext = "0.2" # Prints error contexts on crashes
 ehttp = "0.5.0"
 enumset = "1.0.12"
 env_logger = { version = "0.10", default-features = false }
-ffmpeg-sidecar = "1.1.2"
+ffmpeg-sidecar = { version = "2.0.1", default-features = false }
```

why are we using a legacy version? :)
```suggestion
ffmpeg-sidecar = { version = "2.0.2", default-features = false }
```

---

lol, didn't expect this to go so fast :D